packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -444,9 +428,14 @@ export async function* OpenAIChatModel(
     return next.value.choices[0].delta;
   }
 
+  let isAssistant = false;
```

Comment:
Why is this now necessary?

Reply:
I discovered that previously we were emitting:

```tsx
<AssistantMessage></AssistantMessage>
<FunctionCall name="foo" />
```

So I added this check to wait for any content tagged with the assistant role.